### ISSUE TITLE
Missing space before inline math

### ISSUE THREAD

**Issue by abdullahkhalids** [cqql/pelican-katex#4]
I have a simple pelican page, whose source is

```
Title: Katex test
Date 2020-02-03

This is a math test $E=mc^2$. Are spaces being properly formatted? 
Is there a space before the math? What about for this  $E=mc^2$?
```

Rendered at https://abdullahkhalid.com/katex-test/ 

What happens: For the first equation there is only one space before the $ sign, and this space is eaten up in the rendered version. For the second equation, I have inserted a additional space before the $ sign, which then results in a space before the inline math in the rendered result. 
![Screenshot_2020-02-03_00-46-48](https://user-images.githubusercontent.com/2965278/73614187-b2be4a80-461e-11ea-95ca-94987638e273.png)

Expected result: The space before inline math should not be eaten up.

**Comment by cqql** [comment]
Thanks for the report! I just released version 1.3.1 that handles preceding whitespace correctly.

**Comment by abdullahkhalids** [comment]
Thanks. Can confirm that whitespace is being rendered correctly now.
